libs/platform/src/lib/components/form/input/input.component.ts

```diff
@@ -58,6 +58,9 @@ export class InputComponent extends BaseInput implements OnInit, AfterViewInit {
     @Input()
     type: InputType = 'text';
 
+    @Input()
+    required: boolean;
```

Comment:
can we not have this in `base-input` file instead?

Reply:
required is input property in form-field component, from there while registering other form components, required field is set. But in case of Input component, as it is being used by other component, from parent component; required field need to be passed.

If we put required as input property in BaseInput, then it will be for all form-components. so field-component required input field will loose the mean.

Reply:
I see.. thanks for explaining.